loadouts/cargo-list.sqf

```diff
@@ -130,4 +130,13 @@ switch(_x) do {
             ["R3F_FUM40", 40]
         ]] call BRM_FMK_fnc_addtoCargo;
     };
+    
+    case "SMAW":{
```

Comment:
This isn't a good solution to the problem. Should also have also been a separate PR.